ios/Lift/LiveSessionController.swift

```diff
@@ -136,14 +137,31 @@ class LiveSessionController: UITableViewController, UITableViewDelegate, UITable
     
     override func tableView(tableView: UITableView, didSelectRowAtIndexPath indexPath: NSIndexPath) {
         if indexPath.section == 1 {
-            NSLog("Explicitly classification start")
+            if let selectedCell = tableView.cellForRowAtIndexPath(indexPath) {
+                let cellText = selectedCell.textLabel!.text
```

Comment:
You might find it easier to use ``exampleExercises[indexPath.row]``, which gives you ``Exercise.Exercise`` directly, you don't have to construct it from the cell's text.